pkg/printer/printer.go

```diff
@@ -220,18 +235,21 @@ func (p *Printer) WriteProtoFlow(res *observerpb.GetFlowsResponse) error {
 			return fmt.Errorf("failed to write out packet: %v", err)
 		}
 	case DictOutput:
+		var err error
```

Comment:
This is shadowing `err` declared at line 195. Because of that, the `err` check at line 259 will always take the false branch.

Note that this `err` is actually used by line 243.